src/Tizen.NUI/src/internal/Layouting/LayoutController.cs

```diff
@@ -107,10 +105,10 @@ public Animation GetCoreAnimation()
         /// </summary>
         internal void CreateProcessCallback()
         {
-            if (instance == null)
+            if (LayoutProcessorCallback == null)
             {
-                instance = new Callback(Process);
-                Interop.LayoutController.SetCallback(SwigCPtr, instance);
+                LayoutProcessorCallback = new ProcessorInterface.ProcessorCallback(Process);
```

Comment:
I think layout's processor need to be considered the last processor of all NUI processor...
we can devide pre-process event and process event...

Reply:
Currently we don't need to post process of layouting.
I think we can add post process when we need. thank you.